libraries/AP_OSD/fonts/Param-Tokens-Plane.md

```diff
@@ -0,0 +1,2115 @@
+|Key|PKey|Index|Group|Name|Value|
```

Comment:
this will get out of date pretty quickly, better to not include I think

Reply:
Removed